tests/test_spag.py

```diff
@@ -514,6 +517,53 @@ def test_spag_template_shortcut(self):
         self.assertEqual(json.loads(out), {"id": "wumbo"})
         self.assertEqual(ret, 0)
 
+    def test_spag_template_default(self):
+        _, err, ret = run_spag('request', 'template/post_thing',
+                               '--with', 'thing_id=mydefaultid')
+        self.assertEqual(err, '')
+        self.assertEqual(ret, 0)
+
+        out, err, ret = run_spag('request', 'template/get_default')
+        self.assertEqual(err, '')
+        self.assertEqual(json.loads(out), {"id": "mydefaultid"})
+        self.assertEqual(ret, 0)
+
+    def test_spag_template_from_default_and_active_environments(self):
```

Comment:
They do too work. You just need the quotes to have the space in the value, which is done for you by `subprocess.call` or `Popen` or whatever it is.

    spag env set 'mini=barnacle boy' 'wumbo=mermaid man'